packages/api/src/moderation/subjects/profile.ts

```diff
@@ -0,0 +1,31 @@
+import { ModerationCauseAccumulator } from '../accumulator'
+import {
+  Label,
+  ModerationSubjectProfile,
+  ModerationOpts,
+  ModerationDecision,
+} from '../types'
+
+export function decideProfile(
```

Comment:
Are there times that we want to decide on the profile as something different from the account? 

I'm fine with having them split out for now but it's just one extra function to juggle & it seems like we'll want to consider the profile any time we're looking at an account (since it provides the display name & avatar).
It seem

Reply:
I believe @bnewbold surfaced some fairly common scenarios where this comes-up, and that it's designed this way to support them.

Reply:
ah cool cool 👍

Reply:
Yep this is actually a very specific choice that's used actively